charts/kube-otel-stack/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: kube-otel-stack
 description: Chart for sending Kubernetes metrics to Lightstep using the OpenTelemetry Operator.
 type: application
-version: 0.2.8
+version: 0.2.9
```

Comment:
Heads up that Jacob just merged a pr so this should be bumped before merging

Reply:
Thanks, just updated to `0.2.10`